docs/wiki/user/radio-signal-loss.md

```diff
@@ -51,7 +51,7 @@ API function example:
 
 `true` enables the full duplex mode.
 
-###  Ignore Antenna direction (New setting to be introduced in v2.2.0)
+###  Ignore Antenna direction (introduced in v2.2.0)
```

Comment:
I thought of streamlining it in this PR, but it has to be streamlined across all files, so that'll be in another PR (if I'll make one).